### PR TITLE
ETSI 512 1.1.2 Update

### PR DESCRIPTION
**Pull request by sake**
This pull request updates the 512 schema to version 1.1.2

### CODE REVIEW

**Review by ervta — APPROVED** [review]
seems to be ok. we should take care to not have too many format driven changes, as this time.

**Review by ervta — APPROVED** [review]

File: service/src/main/resources/config/profile.xml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 	<!-- ================================================ -->
-	<!-- Version of 05.07.2020                            -->
+	<!-- Version of 02.09.2020                            -->
 	<!-- ================================================ -->
 
-	<md:ProfileIdentifier>http://www.bsi.bund.de/tr-esor/V1.2.2/profile/preservation-api/V1.1.1</md:ProfileIdentifier>
+	<md:ProfileIdentifier>http://www.bsi.bund.de/tr-esor/V1.2.2/profile/preservation-api/V1.1.2</md:ProfileIdentifier>
 
 	<md:Specification>https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR03125/BSI_TR_03125_Anlage_E_V1_2_2.pdf</md:Specification>
 	<md:Specification>https://www.etsi.org/deliver/etsi_ts/119500_119599/119512/01.01.01_60/ts_119512v010101p.pdf</md:Specification>
```
Comment (ervta) [inline service/src/main/resources/config/profile.xml]: the version of the sepcification should be adjusted as well.

**Review by ervta — CHANGES_REQUESTED** [review]
the version of the specification in the profile is still 1.1.1

File: service/src/main/resources/config/profile.xml (hunk 1 above)
Comment (sake) [reply service/src/main/resources/config/profile.xml]: I initiated the discussion whether the ProfileIdentifier shall be updated. The Result will be reflected here as soon as it is clear how to proceed.

**Review by ervta — APPROVED** [review]

**Pull request merged**